addons/test_discuss_full/tests/test_performance.py

```diff
@@ -103,11 +103,13 @@ def test_init_messaging(self):
         with self.assertQueryCount(emp=82):
             init_messaging = self.users[0].with_user(self.users[0])._init_messaging()
 
+        group_fullname = self.env.ref('base.group_user').full_name
```

Comment:
I would just store `group_user` in the variable to avoid the `ref`, storing the name is a bit too restrictive (what if we need something else?) and makes the naming harder (I don't like current name, which group? etc).